ovos_plugin_manager/vad.py

```diff
@@ -44,12 +44,13 @@ def get_class(config=None):
         The configuration file ``mycroft.conf`` contains a ``vad`` section with
         the name of a VAD module to be read by this method.
 
-        "vad": {
+        "VAD": {
             "module": <engine_name>
         }
         """
         config = get_vad_config(config)
-        vad_module = config.get("module", "dummy")
+        config.setdefault("module", "dummy")
```

Comment:
should we load dummy? i think it makes sense to throw an error since this is an essential piece

Reply:
We do the same for hotwords; I think it is helpful to enable loading modules without optional components like VAD and Hotwords i.e. for testing or applications where you only need the STT component. Alternatively, this exception could be caught in the caller (listener)

Reply:
i think we should raise exceptions since its impossible to work without this. testing or applications need to account for this explicitly some other way

we can  work without hotwords, but better error handling around that is also needed

Reply:
The non-dinkum listener will work without VAD. That can be left to configuration though, this was mainly intended to fix what looked to be the intended default

Reply:
let's start making the factories throw exceptions and let the individual services decide if they want dummy or not